### PR TITLE
Ch postgres

### PR DESCRIPTION
**Pull request by jemmyarina**
What does this PR do?
This PR add a PR request for merging with DEVELOP branch 

Description of Task to be completed?
POSTGRES CONFIGURATION AND SEQUELIZE SETUP

How should this be manually tested?
IT SHOULD BE TESTED MANUALLY USING PGADMIN 4

Any background context you want to provide?
N/A

What are the relevant pivotal tracker stories?
#175743303

What are the relevant Github Issues?
N/A

Screenshots (if appropriate)
N/A

Questions:
N/A

### CODE REVIEW

**Review by Anguandia — CHANGES_REQUESTED** [review]
Please work on the following:
- PR description
- Add node modules to git ignore
- Rebase and resolve conflicts

**Review by Anguandia — CHANGES_REQUESTED** [review]
Add node_modules to git ignore
Improve the PR description
Rebase and resolve conflicts
Always squash multiple commits

**Review by leny62 — CHANGES_REQUESTED** [review]
The package-lock.json is presenting many conflicts. So you may delete it, rebase and reinitialise/reinstall your npm.

**Review by furebo — COMMENTED** [review]
The PR description  is about the configuration of postgres and set up of sequelize

**Review by leny62 — APPROVED** [review]
Good Job @jemmyarina , Let's now see whether this PR isn't conflicting with the base branch.

**Review by furebo — APPROVED** [review]
Thank you for solving conflicts issue

File: .babelrc [hunk 1]
```diff
@@ -1,8 +1,13 @@
 {
+<<<<<<< HEAD
```
Comment (Anguandia) [inline .babelrc]: You didn't finish with the process of rebasing/resolving conflicts, please complete this process

File: .babelrc [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
   "presets": ["@babel/preset-env"],
   "plugins": [
     [
       "@babel/plugin-transform-runtime"
     ]
   ]
+=======
```
Comment (Anguandia) [inline .babelrc]: Same here

File: .babelrc [hunk 3]
```diff
@@ -1,8 +1,13 @@
...
+    "presets" : ["@babel/preset-env"],
+    "plugins" : ["@babel/plugin-transform-runtime"]
+>>>>>>> index and config
```
Comment (Anguandia) [inline .babelrc]: and here

File: .hound.yml [hunk 4]
```diff
@@ -1,5 +1,9 @@
 jshint:
+<<<<<<< HEAD
   enabled: false
 
 jshint:
   ignore_file: .jshintignore
+=======
+    enabled: false
+>>>>>>> adding seeds successfully
```
Comment (Anguandia) [inline .hound.yml]: Complete the rebasing please

**Review by Anguandia — CHANGES_REQUESTED** [review]

**Review by Monehin — COMMENTED** [review]
Fix the Deepcode Issue

Just follow the example from the [JsDocs](https://jsdoc.app/tags-param.html#:~:text=The%20%40param%20tag%20provides%20the,a%20description%20of%20the%20parameter) Documentation to define the model's params

**Review by Anguandia — APPROVED** [review]

**Pull request merged**